README.md

```diff
@@ -1,12 +1,8 @@
 ![conduit][logo]
 
+[![Build Status](https://travis-ci.org/runconduit/conduit.svg?branch=master)](https://travis-ci.org/runconduit/conduit)
```

Comment:
For consistency with the other badges, I have a slight preference for moving these links to the refs section at the bottom of this document, and then referencing them by their labels here.